playbooks/ansible/roles/sit.gpfs/tasks/new_share.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Bind the GPFS volume to mount path
+  mount:
+    path: "{{ config.paths.mount }}/{{ volume }}"
+    src: "/gpfs/sit_fs/{{ volume }}"
+    opts: bind
+    fstype: none
+    state: mounted
+
+- name: Set permissions to root directory
+  file:
+    path: "{{ config.paths.mount }}/{{ volume }}"
+    mode: 0777
```

Comment:
Isn't this already part of _fix_share_permissions.yml_ above?

Reply:
Yes. I'll remove it.